src/metabase/channel/slack.clj

```diff
@@ -325,10 +325,12 @@
    text-or-nil :- [:maybe :string]
    & [message-content]]
   ;; TODO: it would be nice to have an emoji or icon image to use here
-  (let [base-params {:channel     channel-id
-                     :username    "MetaBot"
-                     :icon_url    "http://static.metabase.com/metabot_slack_avatar_whitebg.png"
-                     :text        text-or-nil}
+  (let [base-params {:channel      channel-id
+                     :username     "MetaBot"
+                     :icon_url     "http://static.metabase.com/metabot_slack_avatar_whitebg.png"
+                     :text         text-or-nil
+                     :unfurl_links false
+                     :unfurl_media false}
```

Comment:
what does this do? don't we want to show images when part of subscriptions?

Reply:
The last time I checked, we already didn't expand them automatically. 🤷‍♂️ 

Besides, check the screenshots in the description.
See that the "Best selling products" (79kb) is collapsed?

Reply:
I have the same question - seems like maybe we want `unfurl_links` false but `unfurl_media` true?

Reply:
I couldn't stop the link from unfurling no matter what. At least, not reliably.

The new approach is to remove the link.
I'll mark the convo as resolved since it became outdated.